docs/releases/release120.rst

```diff
@@ -0,0 +1,30 @@
+Release 1.2.0
+=============
+
+Date: 15th Dec, 2022
+
+Overview
+--------
+
+Klaw version 1.2.0 is a minor release with a few features and improvements.
+
+New features
+------------
+
+- Introducing React UI:
+    React is a popular open-source JavaScript library that simplifies the development of front-end applications and provides a great alternative to large, complex front-end frameworks like AngularJS. Klaw's current interface, which is built with AngularJS, does not leverage any modern layouts or components and has several limitations in terms of implementations and reusable components.
+    React allows for greater flexibility and modularity, making it easier to create reusable components and scale an application. Additionally, with React, it's easier to create consistent and cohesive user experiences.
+    Moving to React for Klaw's UI provides numerous benefits in terms of performance, scalability, and ease of development. We've tested React with our current interface, and it looks amazing - we're excited to start this new journey.
+
+- New look and feel for Browse topics:
+    This release introduces the redesigned user interface for Browse topics, providing a new look and feel for managing Kafka topics. It consists of changes to the layout, styling, and overall appearance of the interface, as well as improvements to make browsing and working topics easier and more efficient. This new look and feel is achieved by switching to React.
```

Comment:
Same comment as above, I feel we can add "accessibility" here. It's something that may be important for some potential users (single users but also institutions) and it's also something we can show off a bit!

Reply:
Updated. Pls take another look.